src/main/server.ts

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+import * as logging from "@hmcts/nodejs-logging";
+import * as fs from "fs";
+import * as https from "https";
+import * as path from "path";
+import { app } from "./app";
+
+const logger = logging.getLogger("server");
+
+// TODO: set the right port for your application
+const port: number = parseInt(process.env.PORT, 10) || 3100;
```

Comment:
remove `10`

Reply:
done

Reply:
As discussed, added that argument back in order to make it implementation-independent.